api/admin/controller.py

```diff
@@ -1906,4 +1908,34 @@ def discovery_services(self):
         else:
             return Response(unicode(_("Success")), 200)
 
+    def library_registrations(self, do_get=HTTP.get_with_timeout, do_post=HTTP.post_with_timeout):
+        if flask.request.method == "POST":
+
+            integration_id = flask.request.form.get("integration_id")
+            library_short_name = flask.request.form.get("library_short_name")
+
+            integration = get_one(self._db, ExternalIntegration,
+                                  goal=ExternalIntegration.DISCOVERY_GOAL,
+                                  id=integration_id)
+            if not integration:
+                return MISSING_SERVICE
+
+            library = get_one(self._db, Library, short_name=library_short_name)
+            if not library:
+                return NO_SUCH_LIBRARY
+
+            auth_document_url = self.url_for("acquisition_groups", library_short_name=library.short_name)
```

Comment:
We could store the registration link instead of the main registry link, or we could get the feed and find the register link when we first create the integration.. Would either of those be better?